runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -886,12 +884,11 @@ public MessageConsumer newStream(
             countRequests.getAsLong();
             this.versions = versions;
 
-            final BeginFW begin = beginRO.wrap(buffer, index, index + length);
             HttpClient client = supplyClient();
 
             MessageConsumer newStream = null;
 
-            final int extensionLength = begin.extension().sizeof();
+            final int extensionLength = begin.extension().sizeof() + 3 * Long.BYTES + Integer.BYTES;
```

Comment:
The terminology has shifted slightly, so we need to rename these variables for clarity.
This is no longer `extensionLength`, it is now `queuedRequestLength` and `maximumHeadersSize` is `maximumRequestQueueSize`, agree?

Reply:
Yep